Yank/experiment.py

```diff
@@ -3090,7 +3097,27 @@ def _build_experiment(self, experiment_path, experiment, use_dummy_protocol=Fals
     # Experiment run
     # --------------------------------------------------------------------------
 
-    def _run_experiment(self, experiment):
+    def _write_status_file(self, experiment_path):
```

Comment:
Hmm. You may have to wrap this in the `@mpi.on_single_node(rank=0)` decorator, or with MPI we'll have a bunch of processes trying to access the netcdf file and write the same pickle file.

Reply:
Will this still work even if MPI is not in use?

Reply:
Yes, the `mpi` module should take care of just running serially if it detects that MPI is not being used.